documents/design/principles.md

```diff
@@ -58,6 +58,23 @@ ICU4C/ICU4J exposes certain pieces of data through user-facing APIs such as Date
 
 Runtime customizability of locale data can sometimes come at a performance or memory cost.
 
+## Locale data from multiple sources works seamlessly
+
+*What:* If data is available for a particular constructor and locale, the resulting behavior should not change based on where the data was sourced, with a narrow exception for data that primarily impacts performance characteristics.
```

Comment:
this basically says that behaviour can never change even if CLDR data changes. If I'm source from more recent CLDR, behaviour *should* often change

Reply:
Yeah good point; that obviously wasn't the intention. Is this better?